server/app/controllers/customer_card.py

```
@@ -0,0 +1,74 @@
+from abc import ABC
+from typing import Literal, TypedDict
+
+from ..dal.repositories.customer_card import CustomerCardRepository
+from ..dal.schemas.customer_card import (
+    CustomerCard,
+    CustomerCardCreate,
+    CustomerCardUpdate,
+)
+
+
+class BaseCustomerCardController(ABC):
```

Ruff (0.11.9)

`BaseCustomerCardController` is an abstract base class, but it has no abstract methods or properties

(B024)